ZBUG-866 : Fixing message content missing in chrome73

BhavinBathani opened the pull request:
As described in ticket that src attribute of message-body-iframe
with "javascript:\" is preventing the content to be loaded at all.
Avoiding the practice to set any kind of url in src attribute to
iframe.

BhavinBathani committed:
ZBUG-866 : Fixing message content missing in chrome73

As described in ticket that src attribute of message-body-iframe
with "javascript:\" is preventing the content to be loaded at all.
Avoiding the practice to set any kind of url in src attribute to
iframe.

silentsakky commented on WebRoot/js/ajax/dwt/widgets/DwtIframe.js:
```diff
@@ -261,7 +261,7 @@ DwtIframe.prototype._createFrame = function(html) {
 		if(self._onLoadHandler){
 			tmp[i++] = " onload='" + self._onLoadHandler + "'";
 		}
-		tmp[i++] = " src='javascript:\"\";' ></iframe>";
```
instead of removing src, can we set it to about:blank? as per https://www.w3.org/TR/2011/WD-html5-20110525/fetching-resources.html#about:blank, content will be immediately available

BhavinBathani replied:
Fixed. and working fine as far as i check.

BhavinBathani committed:
ZBUG-866 : Fixing review comments.

Using about:blank to define empty url instead javascript://
which doesn't prevented by Chrome-73 version.
